examples/src/main/scala/com/criteo/cuttle/examples/HelloWorld.scala

```diff
@@ -45,35 +45,38 @@ object HelloWorld {
           val partitionToCompute = (e.context.start) + "-" + (e.context.end)
 
           // The `sh` interpolation is provided by the local platform. It allows us to
-          // declare a bash script to execute.
-          sh"""
-          echo "Hello for ${partitionToCompute}"
-          echo "Check my project page at https://github.com/criteo/cuttle"
-          sleep 1
-        """.exec()
+          // declare a bash command to execute. If you want use many commands you can combine them
+          // in a for comprehension.
+          for {
+            _ <- exec"echo Hello for $partitionToCompute" ()
+            _ <- exec"echo Check my project page at https://github.com/criteo/cuttle" ()
+            completed <- exec"sleep 1" ()
```

Comment:
Again I don't think it is a good idea to for a process to log a message. People will probably do the same in their own project by copying the example. Instead use a singe exec""() and replace the echo by pure scala code doing the logging (e.streams.info(...))